pkg/profile/profile.go

```diff
@@ -2,60 +2,29 @@
 package profile
 
 import (
-	"github.com/redhat-developer/app-services-cli/internal/config"
-	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"os"
+
+	"github.com/spf13/cobra"
 )
 
-// Visual element displayed in help
-func ApplyDevPreviewLabel(message string) string {
-	return "[Preview] " + message
-}
+var DevPreviewEnv string = "RHOAS_DEV"
 
-// Annotation used in templates and tools like documentation generation
-func DevPreviewAnnotation() map[string]string {
-	return map[string]string{"channel": "preview"}
-}
+// ApplyDevPreviewLabel adds visual element displayed in help
+func ApplyDevPreviewLabel(cmd *cobra.Command) {
+	cmd.Short = "[preview] " + cmd.Short
```

Comment:
Maybe we could highlight this in a different colour?

Reply:
I had that done. That will break ascidocs generation sadly and I could not figure out how to not include colours into docs